packages/core/src/providers/AbstractProvider.ts

```diff
@@ -125,24 +125,32 @@ export type RuneBalanceResponse = {
 
 export type RunesResponse = {
 	/**
-	 * The limit of runes
+	 * The limit of runes (MempoolSpace)
```

Comment:
I belive there should be another way to unify the API. At the moment I would suggest not to extend the API in the AbstractProvider and keep it as it was. There were some poor decisions made for data provider abstraction in the first place and there should be no Indexer of Runes mixed with basic bitcoin data. We will address this soon. 

So, I would suggest to map the API Maestro has to the existing responses so we don't break the lib.